server/src/testFixtures/java/io/crate/testing/Asserts.java

```diff
@@ -259,7 +260,7 @@ public static Function<Scalar, Consumer<Scalar>> isNotSameInstance() {
     public static <T extends Throwable, E extends Throwable> void assertRootCause(T throwable,
```

Comment:
Why do we need this if there is `hasRootCause(cause)` on `AbstractThrowableAssert` ? Can we please remove this again?

Reply:
The issue is with `hasRootCauseMessage()` there is no `hasRootCauseMessageContaining` or starts/endsWith.

Reply:
we can do, yes:
```
assertThatThrownBy(deleteSnapshotFuture::get)
    .extracting(SQLExceptions::unwrap, THROWABLE)
    .isExactlyInstanceOf(ConcurrentSnapshotExecutionException.class)
    .hasMessageContaining(expectedFailureMessage);
```
If you prefer this I can change and remove the method from `Asserts`.

Reply:
Thanks. Yes I think we should stick with existing primitives if possible instead of inventing our own if they'd only save a line at most. We still got several different ways to check for errors (`expectThrows`, `expectedException`) and it would be good reduce them further.